generate-routes.ts

```diff
@@ -184,27 +152,28 @@ const deriveResource = (
   endpointPath: string,
   method: Method,
 ): string | null => {
-  if (isEndpointResource(endpointPath)) {
-    return endpointResources[endpointPath] ?? null
-  }
-
   if (isOpenapiPath(endpointPath)) {
     const spec = openapi.paths[endpointPath]
     const methodKey = method.toLowerCase()
 
     if (methodKey === 'post' && 'post' in spec) {
-      const response = spec.post.responses[200]
-      if (!('content' in response)) return null
-      return deriveResourceFromSchema(
-        response.content['application/json']?.schema?.properties ?? {},
-      )
+      const postSpec = spec.post
+      const openapiResponseKeyProp = 'x-fern-sdk-return-value'
```

Comment:
Can you move this to the very top of the file, around line 8 :)

Reply:
Sure!

Reply:
https://github.com/seamapi/javascript-http/pull/87/commits/08a36f25e2e80a88ec3fe5ed50546df5afe879c8

Reply:
Whoops! I moved it to line 12, but after merging with the main branch, it ended up at line 34. Let me know if this placement is okay or if I should move it closer to the beginning of the file.